### PR TITLE
change template to use new backend system

### PR DESCRIPTION
**Pull request by angeliski**
## Hey, I just made a Pull Request!

Moving cli template to use new backend system (#18301)

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [x] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [n/a] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/tall-cars-boil.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli': patch
```
Comment (angeliski) [inline .changeset/tall-cars-boil.md:2]: Not really sure if this is a patch 🤔

File: packages/cli/templates/default-backend-plugin/src/plugin.ts.hbs [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/import { loggerToWinstonLogger } from '@backstage/backend-common';
```
Comment (adamdmharvey) [inline packages/cli/templates/default-backend-plugin/src/plugin.ts.hbs:1]: Extra prefix slash?

Comment (angeliski) [reply packages/cli/templates/default-backend-plugin/src/plugin.ts.hbs:1]: Just a typo from my other window 😅 Thanks for the catch

**Comment by benjdlambert** [comment]
Hey :wave: thanks for this! I'm not sure we're just there yet with this. I think we're probably going to want to be using this fully before recommending that we start from scratch with this at least. Hopefully though in a few weeks we might be ready for this. Do you mind if we close this and pick it back up when we're ready? @angeliski?

**Comment by angeliski** [comment]
Sure @benjdlambert 
Just got excited with the new system

If you prefer I can restore the exporter for the router too and people will choose with system use

**Comment by benjdlambert** [comment]
I think we would probably just want to make a switch to the new system when we're ready. Thinking that a newcomer to backstage is not going to even know about this new backend system until they've been experienced with Backstage for a while, so not sure an option provides any value here unfortunately. 

Can I close this PR and then we can pick it back up once we're ready to roll this out?  :pray:

**Pull request closed without merging**